website/content/api/stocks/_index.md

```diff
@@ -1,6 +1,67 @@
 ---
-title: stocks
-keywords: ""
-excerpt: ""
+title: Introduction to the Stocks SDK
+keywords: "stocks, stock, options, option, call, put, earnings, calendar, how-to, guide, scripts, fundamental, analysis, technical, behavioural, analyst, equity, research, api, sdk, application, python, notebook, jupyter"
+excerpt: "This guide introduces the Stocks menu in the context of the OpenBB SDK."
 geekdocCollapseSection: true
 ---
+
+The <a href="https://openbb-finance.github.io/OpenBBTerminal/terminal/stocks/" target="_blank">Stocks menu</a> wraps all of the functions from the OpenBB Terminal application into a powerful SDK, enabling users to work with the data in a flexible environment that can be fully customized to meet the needs of any user. Code completion and contextual help makes the SDK is easy to use. Navigation is very similar to operating the CLI Terminal Application. 
+
+## How to use the Stocks SDK
+
+Start a Jupyter Notebook with as little as one line of code:
+
+`from openbb_terminal.api import openbb`</br>
+
+In the next cell, the code complete will activate after the `.` is entered, showing the submenus and functions at the root level of the <a href="https://openbb-finance.github.io/OpenBBTerminal/terminal/stocks/" target="_blank">Stocks menu</a>.
```

Comment:
**Use instead:** In the next cell, code completion will activate after the `.` .......